orchagent/nexthopkey.h

```diff
@@ -22,18 +22,33 @@ struct NextHopKey
     uint32_t            weight;         // NH weight for NHGs
     string              srv6_segment;   // SRV6 segment string
     string              srv6_source;    // SRV6 source address
+    string              srv6_vpn_sid;   // SRV6 vpn sid
 
-    NextHopKey() : weight(0) {}
+    NextHopKey() : weight(0),
+        srv6_vpn_sid(""),
+        srv6_source(""),
+        srv6_segment("")
+    {}
     NextHopKey(const std::string &str, const std::string &alias) :
-        alias(alias), vni(0), mac_address(), weight(0)
+        alias(alias), vni(0), mac_address(), weight(0),
+        srv6_vpn_sid(""),
+        srv6_source(""),
+        srv6_segment("")
     {
         std::string ip_str = parseMplsNextHop(str);
         ip_address = ip_str;
     }
     NextHopKey(const IpAddress &ip, const std::string &alias) :
-        ip_address(ip), alias(alias), vni(0), mac_address(), weight(0) {}
+        ip_address(ip), alias(alias), vni(0), mac_address(), weight(0),
+        srv6_vpn_sid(""),
+        srv6_source(""),
+        srv6_segment("")
+    {}
     NextHopKey(const std::string &str) :
-        vni(0), mac_address()
+        vni(0), mac_address(),
+        srv6_vpn_sid(""),
```

Comment:
can we move the vpn_sid to the end of NH key without affecting the non-VPN NH key?

Reply:
It will not affect non-VPN NH key, however I moved srv6_vpn_sid to end of NH key.